cpp/tests/ast/transform_tests.cpp

```diff
@@ -94,6 +95,31 @@ TEST_F(TransformTest, NullLiteral)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, result->view(), verbosity);
 }
 
+TEST_F(TransformTest, IsNull)
+{
+  auto c_0   = column_wrapper<int32_t>{{0, 1, 2, 0}, {0, 1, 1, 0}};
+  auto table = cudf::table_view{{c_0}};
+
+  auto literal_value = cudf::numeric_scalar<int32_t>(-123);
+  auto literal       = cudf::ast::literal(literal_value);
+  auto expression    = cudf::ast::operation(cudf::ast::ast_operator::IS_NULL, literal);
+
+  auto result    = cudf::compute_column(table, expression);
+  auto expected1 = column_wrapper<bool>({0, 0, 0, 0});
+  CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected1, result->view(), verbosity);
+
+  literal_value.set_valid_async(false);
+  result         = cudf::compute_column(table, expression);
+  auto expected2 = column_wrapper<bool>({1, 1, 1, 1}, cudf::test::iterators::no_nulls());
+  CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected2, result->view(), verbosity);
```

Comment:
This could use a comment indicating that even though you're executing an operation over the column, the data in the table is irrelevant aside from its size because the expression embeds the literal.

Reply:
added comment.